src/main/java/graphql/testrunner/repository/TestResultRepo.java

```diff
@@ -0,0 +1,43 @@
+package graphql.testrunner.repository;
+
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TestResultRepo {
+
+    public static final Logger LOGGER = Logger.getLogger(TestResultRepo.class.getName());
+    private static final String COLLECTION_NAME = "test-runs";
+
+    @Autowired
+    private Firestore firestore;
+
+    public void updateInitialTestResult(UUID jobId, Map<String, Object> updates) {
+        updateDocument(jobId, updates);
```

Comment:
these functions do the same thing, and take the same parameters, do they need to be seperate?

Reply:
done